includes/attendee-repository.php

```diff
@@ -37,25 +43,45 @@ public function remove_attendee( int $event_id, int $user_id ): void {
 			throw new Exception( 'invalid user id' );
 		}
 
-		$event_ids = get_user_meta( $user_id, self::USER_META_KEY, true );
-		if ( ! $event_ids ) {
-			$event_ids = array();
-		}
-
-		if ( isset( $event_ids[ $event_id ] ) ) {
-			unset( $event_ids[ $event_id ] );
-		}
-
-		update_user_meta( $user_id, self::USER_META_KEY, $event_ids );
+		global $wpdb, $gp_table_prefix;
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
+		$wpdb->query(
+			$wpdb->prepare(
+				"delete from {$gp_table_prefix}event_attendees where event_id = %d and user_id = %d",
+				array(
+					'event_id' => $event_id,
+					'user_id'  => $user_id,
+				),
+			),
+		);
+		// phpcs:enable
 	}
 
 	public function is_attending( int $event_id, int $user_id ): bool {
-		$event_ids = get_user_meta( $user_id, self::USER_META_KEY, true );
-		if ( ! $event_ids ) {
-			$event_ids = array();
-		}
+		global $wpdb, $gp_table_prefix;
+
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.NoCaching
```

Comment:
Could you check again if these are needed?

Reply:
I have confirmed that all these disables are needed, also the ones you mentioned in other comments.

The `InterpolatedNotPrepared` is because of the table name interpolation.